add.php

```diff
@@ -114,7 +112,15 @@ function submit() {
             var email     = $("#email-input").val();
             var id        = $("#id-input").val();
             var enterer   = $("#enterer-input").val();
-            var committee = $("#id-committee").val();
+
+            var committees_total = document.getElementsByClassName("committees-checkbox");
+            var committee = "";
+            for(var i = 0; i < committees_total.length; ++i) {
+                if(committees_total[i].checked) {
+                    committee = committee.concat(", "  + committees_total[i].value);
```

Comment:
Dang really wish we could have python `", ".join(committee.value for committee in committees_total if committee.checked)`

Reply:
I like this option of being able to select multiple committees.  I had to manually limit the committees to 1 while uploading the CSV manually to make it consistent with what the interface would have input. For the sake of legislation and the like, it'd be great to have a "view by committee" so an officer can see their entire roster.

Reply:
If you go to the view page, you can search the table and it'll filter by your query. The only issue is that it doesn't let you search only in one column so the results can have some things you didn't intend. Ex: Someone named Bob Roven might show up when searching ROV. But this is a small issue

Reply:
I don't think that should be a huge problem, most committees are unlikely to be similar to names.